acceptance/ui/features/applications.feature

```diff
@@ -0,0 +1,95 @@
+@apps
+Feature: Application Management
+  In order to use Control Center
+  As a CC admin user
+  I want to manage applications
+
+  @login-required
+  Scenario: View default applications page
+    When I am on the applications page
+    Then I should be on the applications page
+
+  @login-required
+  Scenario: View application template dialog
+    When I am on the applications page
+      And I click the Add-Application Template button
+    Then I should see "Add Application Template"
+      And I should see "Application Template File:"
+
+  @login-required
+  Scenario: View services map
+    When I am on the applications page
+      And I click the Services Map button
+    Then I should not see "Deployment ID"
```

Comment:
We should be able to include some kind of positive verification for viewing the service map. At a minimum, every system should have a service map that contains a node named "Internal Services".  Instead of looking for just any label containing "Internal Services", we need to look for one inside the services map.  The services map is rooted under an `svg` element.

So maybe a step that says something like:
```
And I should see "Internal Services" in the Services Map
```

Conceptually, the implementation of that step would be something like find the `tspan` element which is a child of an `svg` element and the contents of the `tspan` element are "Internal Services" (or whatever label is specified by the caller of the step)